src/applications/vaos/actions/appointments.js

```diff
@@ -129,21 +143,20 @@ async function getClinicDataBySystem(facilityClinicListMap) {
 async function getAdditionalFacilityInfo(futureAppointments) {
   // Get facility ids from non-VA appts or requests
   const requestsOrNonVAFacilityAppointments = futureAppointments.filter(
-    appt => !appt.clinicId,
+    appt => !getVARClinicId(appt),
```

Comment:
This check is really just a check to filter out direct scheduled, in person, appointments. You could probably remove anything that has a `vaos.videoType` and where `vaos.isCommunityCare` is true, instead of parsing out the clinic id.